data/test/scenarios/wml_tests/ScenarioWML/EventWML/events-test_filters.cfg

```diff
@@ -239,42 +238,62 @@
     [/event]
     [event]
         name=attack
+        first_time_only=no
         [filter_attack]
             damage=5
         [/filter_attack]
         {ASSERT ({VARIABLE_CONDITIONAL side_number equals 1})}
-        {VARIABLE_OP triggers add 1}
+        {VARIABLE_OP triggers_on_alice_attack add 1}
     [/event]
     [event]
         name=attack
+        first_time_only=no
         [filter_attack]
             damage=9
         [/filter_attack]
         {ASSERT ({VARIABLE_CONDITIONAL side_number equals 2})}
-        {VARIABLE_OP triggers add 1}
+        {VARIABLE_OP triggerson_on_bob_attack add 1}
```

Comment:
Typo

Reply:
Good catch, and it pointed to some `RETURN` statements that needed to be changed to `ASSERT` because it was only checking the first one.